TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/dto/SocketMessageDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.example.travelpointsbackend.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class SocketMessageDTO {
+    @NotNull(message = "User cannot be null")
+    private Long userId;
+    @NotNull(message = "Name product cannot pe null")
+    private String nameAttraction;
+    @NotNull(message = "Offers cannot be null")
+    private String offers;
```

Comment:
W: for all the messages in all the classes, it would be a nicer idea to create an utility class named ErrorMessages and add there all those strings as constants